.circleci/config.yml

```diff
@@ -469,12 +482,16 @@ jobs:
     working_directory: ~/aeternity
     steps:
       - checkout
+      - *restore_macos_package_cache
       - run:
           name: Install required tools
           command: |
-            brew update
```

Comment:
I see this removes [`update`](https://docs.brew.sh/Manpage#update-options) while not pinning libsodium to a specific version. Could this cause old (insecure?) version to unexpectedly be used in package? I suggest to either keep the update, or pin the version.